lib/yardstick/v2_client/remote_model.rb

```diff
@@ -40,7 +40,7 @@ def instance_uri
 
       module ClassMethods
         def find(token, id)
-          from_api(get("#{resource_uri}/#{id}", query: { token: token }))
+          from_api(get("#{resource_uri}/#{id}", query: { token: token }), token: token)
```

Comment:
wat?

What is this doing? Looks like it's creating the URL with the token in the query string, and then passing the token in again?